becquerel/parsers/spectrum_file.py

```diff
@@ -134,9 +134,3 @@ def energy_to_channel(self, energy):
         """Invert the energy calibration to find the channel(s)."""
         energy = np.array(energy, dtype=float)
         return interp1d(self.energies, self.channels)(energy)
-
-    def bin_width(self, channel):
-        """Calculate the width of the bin in keV at the channel(s)."""
-        en0 = self.channel_to_energy(channel - 0.5)
-        en1 = self.channel_to_energy(channel + 0.5)
-        return en1 - en0
```

Comment:
I don't think we should remove this. This may be used by some other analysis code someone is using and would break that usage. We should think about refactoring in `1.0.0`